rocAL/source/api/rocal_api_data_loaders.cpp

```diff
@@ -1644,9 +1644,9 @@ rocalVideoFileSource(
         DecoderType decoder_type;
         find_video_properties(video_prop, source_path, file_list_frame_num);
         if (rocal_decode_device == RocalDecodeDevice::ROCAL_HW_DECODE)
-            decoder_type = DecoderType::FFMPEG_HARDWARE_DECODE;
+            decoder_type = DecoderType::ROCDEC_VIDEO_DECODE;
```

Comment:
If rocDecode is not available, shouldn't we use ffmpeg HW decoder?

Reply:
Currently we don't check if rocDecode is available and enable it, instead we check for that package
@shobana-mcw Please let me know if we need to introduce a CMake flag to enable the rocDecode